squid/setup.py

```diff
@@ -0,0 +1,88 @@
+# Always prefer setuptools over distutils
+from setuptools import setup
+# To use a consistent encoding
+from codecs import open
+from os import path
+
+import simplejson as json
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+runtime_reqs = ['datadog-base']
+with open(path.join(here, 'requirements.txt'), encoding='utf-8') as f:
```

Comment:
This is actually out-of-date, because Trishank added some additional stuff to the requirements files for security. Can you take a look at his work here: https://github.com/DataDog/integrations-core/pull/934

I'll actually have to make additional changes (because I don't think we cover everything we need to cover), but could you please update this so it looks like the others?